### ISSUE TITLE
Crash on dedicated server

### ISSUE THREAD

**Issue by TheEpicBlock** [TelepathicGrunt/Bumblezone-Fabric#6]
https://github.com/TelepathicGrunt/Bumblezone-Fabric/blob/master/src/main/java/net/telepathicgrunt/bumblezone/features/CaveSugarWaterfall.java#L35
`Block#getName` is marked as client only. Therefor this causes a `NoSuchMethodError` when run on a dedicated server. This code would also break if other languages would be added :P

**Comment by TheEpicBlock** [comment]
https://github.com/TelepathicGrunt/Bumblezone-Fabric/blob/8503fd7a9c53d03ab066940baf45e93c843c66a0/src/main/java/net/telepathicgrunt/bumblezone/entities/PlayerTeleportation.java#L156
Found it being used here too. The translation key for the block should probably be used here

**Comment by TelepathicGrunt** [comment]
Funny story about that lol. In forge (where I made Bumblezone originally for), blocks have a way to get their registry name directly from the block itself due to a Forge patch. In fabric, I didn’t realize this and thought .getName/getLoottableName was the yarn names when it wasn’t. I went to fix that up so it gets registry names again but it seems I missed that one in the sugar water feature. I’ll fix that in about two hours when I can get to my laptop. It’s not suppose to use translatable names as the en_us.json file doesn’t exist on the server which is why I opted to check a block’s registry name to see if they have honeycomb in the name for better mod compat. (I would do tags but that would require me to find every possible mod with honeycombs and they might not even have bees in the name making it impossible to find them)

The second one in player teleportation is actually intentional and shouldn’t crash the server (tested on server locally) because of the player.isClient check. That prevents the server from executing the several client code in that if statement that sends an error message to the player.
